nucliadb_node/src/env.rs

```diff
@@ -0,0 +1,337 @@
+// Copyright (C) 2021 Bosutech XXI S.L.
```

Comment:
I think this is a good suggestion, but it would require doing a careful rewrite that maybe is too much for this PR. It is already too big and would not want to stall its merging longer. I would suggest to create a ticket for it and then implement that rewrite on a separate PR.

Reply:
I already created the ticket some months ago https://app.shortcut.com/flaps/story/2873/rework-configuration-type-on-nucliadb-node